### ISSUE TITLE
Role and state are not defined for checkable items

### ISSUE THREAD

**Issue by zhr1130** [jakezatecky/react-checkbox-tree#276]
In the checkbox tree, role and state are not defined for checkable items and due to this, screen reader only announces name of the item while user navigates on items.

Expected result: Role and state should be defined for items in the checkbox tree so that screen reader user could get an idea of such controls.

![image](https://user-images.githubusercontent.com/18317674/118202074-eedd7f00-b40d-11eb-9f48-a81c1658535c.png)

**Comment by jakezatecky** [comment]
The accessibility tree will now ignore the checkbox icon and focus on the underlying checkbox. This required some styling creativity but makes the underlying JavaScript easier to maintain.
